clang/lib/Basic/LangStandards.cpp

```diff
@@ -10,9 +10,49 @@
 #include "clang/Config/config.h"
 #include "llvm/ADT/StringSwitch.h"
 #include "llvm/Support/ErrorHandling.h"
+#include "llvm/Support/FormatVariadic.h"
 #include "llvm/TargetParser/Triple.h"
 using namespace clang;
 
+const char *clang::languageToString(Language L) {
+  // I would like to make this function and the definition of Language
+  // in the .h file simply expand the contents of a .def file.
+  // However, in the .h the members of the enum have doxygen annotations
+  // and/or comments which would be lost.
```

Comment:
I was trying to figure out whether we should add the notion of a "family" to LangStandards.def but then I realized, that's just the `Language` enumeration in string form. So I think we can drop this comment entirely -- this is a reasonable implementation as-is.